dockers/nvidia/Dockerfile

```diff
@@ -32,25 +32,24 @@ RUN \
         rm -rf pytorch-lightning ; \
         git clone https://github.com/PyTorchLightning/pytorch-lightning.git ; \
         cd pytorch-lightning ; \
-        git checkout ${LIGHTNING_VERSION} ; \
+        git checkout tags/${LIGHTNING_VERSION} ; \
         git submodule update --init --recursive ; \
         cd .. ; \
     fi && \
 # save the examples
-    mv pytorch-lightning/_notebooks notebooks && \
+    mv pytorch-lightning/_notebooks/.notebooks/ notebooks && \
```

Comment:
Is this change correct? I don't see the `.notebooks` directory

Reply:
This directory, isn't it? https://github.com/PyTorchLightning/lightning-tutorials/tree/main/.notebooks

Reply:
yes, you need to switch to `publication` branch with generated notebooks